axum-extra/CHANGELOG.md

```diff
@@ -25,9 +25,13 @@ and this project adheres to [Semantic Versioning].
 - **added:** `WithRejection` extractor for customizing other extractors' rejections ([#1262])
 - **added:** Add sync constructors to `CookieJar`, `PrivateCookieJar`, and
   `SignedCookieJar` so they're easier to use in custom middleware
+- **breaking:** `Resource` has a new `S` type param which represents the state ([#1155])
+- **breaking:** `RouterExt::route_with_tsr` now only accepts `MethodRouter`s ([#1155])
+- **added:** `RouterExt::route_service_with_tsr` for routing to any `Service` ([#1155])
```

Comment:
What happened here? Why do these entries come after some of the link definitions?

Reply:
Not sure what you mean 🤔 

![image](https://user-images.githubusercontent.com/718941/185120811-952b2aab-07e9-4843-ab7f-3ae5108278ac.png)

Reply:
You can click on the filename here to see the diff I was seeing:
![Screenshot_2022-08-17_143057](https://user-images.githubusercontent.com/951129/185123027-6bd4135f-f03e-434f-9939-3bbb9e0a35e5.png)

But looks like it's not an actual problem then 👍🏼

Reply:
Oh yeah I see it when I click the filename. No idea what github is doing.